shared/fs/filepreview/index.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import {globalStyles, globalColors, globalMargins, isMobile} from '../../styles'
+import {Box, BackButton, Button, ClickableBox, Icon, List, Text} from '../../common-adapters'
+import {type IconType} from '../../common-adapters/icon'
+import {pathToIcon} from '../icon'
+import {formatTimeForMessages} from '../../util/timestamp'
+
+
```

Comment:
extra spacing. I assume you don't have eslint integration in your IDE? should catch this / run prettier automatically